pkg/sql/crdb_internal.go

```diff
@@ -337,6 +338,57 @@ CREATE TABLE crdb_internal.tables (
 	},
 }
 
+var crdbInternalTablesTableLastStats = virtualSchemaTable{
+	comment: "the latest stats for all tables accessible by current user in current database (KV scan)",
+	schema: `
+CREATE TABLE crdb_internal.table_row_stats (
```

Comment:
nit -- i'd name this `table_row_statistics`

Reply:
Renamed.